components/game/api/useInitGame.ts

```diff
@@ -1,13 +1,17 @@
 import { useCallback, useState } from "react";
-import { GameId } from "../types";
+import { GameId, PlayerName } from "../types";
 
 export const useInitGame = () => {
   const [loading, setLoading] = useState(false);
-  const init = useCallback(async () => {
+  const init = useCallback(async (PlayerOneName: PlayerName) => {
```

Comment:
var name needs to be in the same style as other code, i.e. camelCase is very usual for js projects